config.py

```diff
@@ -31,21 +31,19 @@ class Config:
     conv_agg_type: str  # "add", "mean", "max", "lstm"
     heterogeneous_prop_agg_type: str  # "sum", "mean", "min", "max", "mul"
     save_model: bool
-    eval_every: int  # (LightGCN) evaluation to run every n epoch
-    lr_decay_every: int  # (LightGCN) lr decay to run every n epoch
-    Lambda: float  # (LightGCN)
+    eval_every: int  # evaluation to run every n epoch
     save_every: float  # How often the model should be saved, Ratio of epochs (eg.: 0.2 * epoch_num)
 
     batch_size: int  # batch size. refers to the # of customers in the batch (each will come with all of its edges)
-    val_split: float
-    test_split: float
     num_neighbors: int  # sample n neighbors for each node for n_hop_neighbors iterations
```

Comment:
Can we leave these in, but instead of defining a ratio of all edges, it would define how many edges to sample per user? eg.: 1,2,3 (int)

Reply:
we don't use this at all, anywhere! we have a completely separate process that deals with it. I don't yet know how to configure it what you want to achieve.

Reply:
I can put it back when we have the mechansim you mentioned. until then, they're very misleading.

Reply:
because we do it in preprocessing right? so the splitting is done before not in real time. I see.

Reply:
actually we do it before preprocessing for fashion, as it's so costly! in the case of movielens, we do it within preprocessing.